chainercv/evaluations/eval_detection_voc.py

```diff
@@ -23,30 +23,33 @@ def eval_detection_voc(
     The code is based on the evaluation code used in PASCAL VOC Challenge.
 
     Args:
-        pred_bboxes (list of numpy.ndarray): A list of :math:`N` bounding
-            boxes. Its index corresponds to an index for the base dataset.
+        pred_bboxes (iterable of numpy.ndarray): An iterable of :math:`N`
```

Comment:
From my understanding, an iterator is an iterable. so `iterable of numpy.ndarray` includes `iterator of numpy.ndarray`.